GitCommands/Git/GitItemStatusConverter.cs

```diff
@@ -2,35 +2,37 @@
 {
     public static class GitItemStatusConverter
     {
-        // const instead of static readonly:
+        // git-status const instead of static readonly:
         // These are external constants not expected to be changed or be used as a standalone library
         public const char AddedStatus = 'A';
         public const char CopiedStatus = 'C';
         public const char DeletedStatus = 'D';
         public const char ModifiedStatus = 'M';
-        public const char IgnoredStatus = '!';
         public const char RenamedStatus = 'R';
-        public const char SkippedStatus = 'S';
+        public const char TypeChangedStatus = 'T';
         public const char UnmergedStatus = 'U';
         public const char UnmodifiedStatus_v1 = ' ';
         public const char UnmodifiedStatus_v2 = '.';
+        public const char IgnoredStatus = '!';
         public const char UntrackedStatus = '?';
 
+        // Unused char, to be used to get a default object
+        public const char UnusedCharacter = '&';
```

Comment:
❔
```suggestion
        public const char NeutralStatus = '&';
```

Reply:
OK
Setting this to draft to await the approval test change